src/app/content/highlights/components/Highlights.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useSelector } from 'react-redux';
+import myHighlightsEmptyImage from '../../../../assets/MHpage-empty-logged-in.png';
+import Loader from '../../../components/Loader';
+import { stripHtmlAndTrim } from '../../hooks/receiveContent';
+import { book as bookSelector } from '../../selectors';
+import { ArchiveTree, ArchiveTreeSection } from '../../types';
+import { stripIdVersion } from '../../utils/idUtils';
+import { summaryHighlights, summaryIsLoading } from '../selectors';
+import { SummaryHighlights } from '../types';
+import * as HStyled from './HighlightStyles';
+import * as Styled from './ShowMyHighlightsStyles';
+
+// tslint:disable-next-line: variable-name
+const Highlights = () => {
+  const book = useSelector(bookSelector);
+  const highlights = useSelector(summaryHighlights);
+  const isLoading = useSelector(summaryIsLoading);
+
+  if (book && Object.keys(highlights).length > 0) {
+    return <Styled.Highlights isLoading={isLoading}>
+      {isLoading ? <Styled.LoaderWrapper><Loader/></Styled.LoaderWrapper> : null}
+      {highlights[book.tree.id]
```

Comment:
there will never be highlights associated with the book id, its always a page id

Reply:
Placeholder interface for `summary.highlights` was `{ [chapterId]: { [pageId]: Highlight[] } }` so for example for Preface I'm setting book's id as a chapterId, but I can change summary.highlights to be just a map with pageIds which should be easier to manage

Reply:
i was worried that re-sorting the pages into their chapters all the time would be annoying and heavy, but we could have chapter OR page ids on the top level of the object, and then have subobject for chapters.

or if you're thinking of something that you think you work well just do it your way and i will look at it.